ModelCatalogueCorePlugin/src/groovy/org/modelcatalogue/core/export/inventory/DataModelToDocxExporter.groovy

```diff
@@ -0,0 +1,417 @@
+        def dataClasses = dataClassService.getTopLevelDataClasses(DataModelFilter.includes(rootModel)).items
+        def builder = new ModelCatalogueWordDocumentBuilder(outputStream)
+
+        List<DataModel> allVersionsOfModel = getAllVersionsOfModel(rootModel, [rootModel])
+
+
+
+        builder.create {
+            document(template: customTemplate) {
+
+                def organisation = rootModel.ext.get(Metadata.ORGANISATION)
+                def owner = rootModel.ext.get(Metadata.OWNER)
+                byte[] imageData
+                if(imagePath) imageData = new URL(imagePath).bytes
+
+                if (imagePath){
+                    paragraph(style: 'headerImage', align: 'right'){
+                        image(data: imageData, height: 1.366.inches, width: 2.646.inches, name: 'frontpageImage')
+                    }
+                }
+
+                if (organisation) {
+                    paragraph(style: 'title', align: 'center') {
+                        text organisation
+                    }
+                }
+
+                paragraph (style: 'subtitle', align: 'center'){
+                    text "${rootModel.name}"
+                }
+
+                paragraph(style: 'document', margin: [top: 120]) {
+                    text "Version ${rootModel.versionNumber} ${rootModel.status}"
+                    lineBreak()
+                    text SimpleDateFormat.dateInstance.format(new Date())
+                }
+                pageBreak()
+                heading1 'Document Management'
+                paragraph(style: 'document', margin: [top: 20]) {
+                    text 'Document Owner: ', font:[bold:true]
+                    text owner
+                    lineBreak()
+                    text 'Authors: ', font:[bold:true]
+                    text "${rootModel.ext.get(Metadata.AUTHORS)}"
+                }
+
+                heading2 'Version Control'
+                table {
+                    row {
+                        cell 'Version', style: 'cell.headerCell'
+                        cell 'Date', style: 'cell.headerCell'
+                        cell 'Summary of Changes', style: 'cell.headerCell'
+                    }
+                    allVersionsOfModel.each { DataModel model ->
+                        row {
+                            cell "${model.versionNumber}", style: 'cell'
+                            cell model.lastUpdated, style: 'cell'
+                            cell "${model.revisionNotes}", style:'cell'
+                        }
+                    }
+                }
+
+                heading2 'Reviewers'
+                table {
+                    row {
+                        cell 'Name', style: 'cell.headerCell'
+                        cell 'Responsibility', style: 'cell.headerCell'
+                        cell 'Date', style: 'cell.headerCell'
+                    }
+                    rootModel.ext.get(Metadata.REVIEWERS).split(',').each { String reviewer ->
```

Comment:
yet another error if the reviewers are `null`. you should expect that any metadata values can be null

```
Cannot invoke method split() on null object
    Line | Method
->>  164 | doCall              in org.modelcatalogue.core.export.inventory.DataModelToDocxExporter$_export_closure3_closure5_closure12
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - 
|     57 | doCall              in com.craigburke.document.core.builder.DocumentBuilder$_invokeMethod_closure1
|     56 | invokeMethod . . .  in com.craigburke.document.core.builder.DocumentBuilder
|    158 | doCall              in org.modelcatalogue.core.export.inventory.DataModelToDocxExporter$_export_closure3_closure5
|     57 | doCall . . . . . .  in com.craigburke.document.core.builder.DocumentBuilder$_invokeMethod_closure1
|     56 | invokeMethod        in com.craigburke.document.core.builder.DocumentBuilder
|    103 | doCall . . . . . .  in org.modelcatalogue.core.export.inventory.DataModelToDocxExporter$_export_closure3
|     57 | doCall              in com.craigburke.document.core.builder.DocumentBuilder$_invokeMethod_closure1
|     56 | invokeMethod . . .  in com.craigburke.document.core.builder.DocumentBuilder
|    102 | export              in org.modelcatalogue.core.export.inventory.DataModelToDocxExporter
|     43 | doCall . . . . . .  in org.modelcatalogue.core.genomics.GenomicsController$_exportGelSpecification_closure2
|    144 | doCall              in org.modelcatalogue.core.AssetService$_storeAssetWithSteam_closure3
|     48 | store . . . . . . . in org.modelcatalogue.core.LocalFilesStorageService
|    138 | storeAssetWithSteam in org.modelcatalogue.core.AssetService
|    200 | doCall . . . . . .  in org.modelcatalogue.core.AssetService$_storeReportAsAsset_closure4_closure5
|     78 | withDefaultAuthorId in org.modelcatalogue.core.audit.AuditService
|    196 | doCall . . . . . .  in org.modelcatalogue.core.AssetService$_storeReportAsAsset_closure4
|     34 | doCall              in grails.plugin.executor.PersistenceContextRunnableWrapper$_run_closure1
|     35 | wrap . . . . . . .  in grails.plugin.executor.PersistenceContextWrapper
|     34 | run                 in grails.plugin.executor.PersistenceContextRunnableWrapper
|    511 | call . . . . . . .  in java.util.concurrent.Executors$RunnableAdapter
|    266 | run                 in java.util.concurrent.FutureTask
|   1142 | runWorker . . . . . in java.util.concurrent.ThreadPoolExecutor
|    617 | run                 in java.util.concurrent.ThreadPoolExecutor$Worker
^    744 | run . . . . . . . . in java.lang.Thread
```